web/html/src/manager/maintenance/details/schedule-details.tsx

```diff
@@ -102,22 +102,24 @@ const MaintenanceScheduleOverview = (props: OverviewProps) => {
           <Column columnKey="right" cell={row => row.right} />
         </Table>
       </BootstrapPanel>
-      <div className="panel panel-default">
-        <div className="panel-heading">
-          <h4>
-            {props.name}
-          </h4>
-        </div>
-        <div className="panel-body">
-          <WebCalendar
-            id={props.id}
-            type={"schedule"}
-            messages={props.onMessage}
-            clearMessages={props.clearMessages}
-            responseError={props.responseError}
-          />
+      { props.calendarName &&
```

Comment:
Is this checking for an empty string or can `calendarName` be undefined too? Right now the prop says there will always be a value, just a heads up it might be out of date. But if we're only checking for an empty string then this is fine.  

One additional comment regarding this prop, from simply reading the code it isn't very clear what the difference between `name` and `calendarName` is?

Reply:
It's actually either a string (if there is a related calendar) or undefined (if there is none).
During my testing this worked fine. Do you think I should explicitly check for undefined so
`props.calendarName !== undefined`

As for the names `name` is in this case the name of the maintenance schedule while `calendarName` is the name of the maintenance calendar

Reply:
This check is fine, if it can be `undefined` then the prop definition should be either `calendarName?: string` or `calendarName: string | undefined`, depending on which case is correct.  

As for `name`, a comment on the prop declaration could help, because personally I got confused. :D

Reply:
Ok makes sense. Updated the PR accordingly and added a comment for the name prop